mytap-runtests.sql

```diff
@@ -46,9 +46,18 @@ BEGIN
             ROUTINE_TYPE = 'PROCEDURE'
             AND ROUTINE_SCHEMA = db_
             AND ROUTINE_NAME LIKE 'test%'
-        ORDER BY ROUTINE_NAME;
+        ORDER BY
+            IF(random_seed IS NULL, NULL, RAND(random_seed))
```

Comment:
There is a slight chance that shuffled order will be same as alphabetical one, but I don't think that's problem.